tests/phpunit/tests/template.php

```diff
@@ -629,6 +629,42 @@ public function data_extra_args_for_add_settings_section() {
 		);
 	}
 
+	/**
+	 * Tests that `locate_template()` uses the current theme even after switching the theme.
+	 *
+	 * @ticket 18298
+	 * @covers ::locate_template
```

Comment:
```suggestion
	 * @ticket 18298
	 *
	 * @covers ::locate_template
```
🔢 If changed, applies elsewhere in the PR.

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5160/commits/7bc339ec9f14f53b1e9777bbc01a1200a760a959